src/EditorFeatures/CSharp/ContentType/ContentTypeDefinitions.cs

```diff
@@ -17,6 +17,10 @@ internal static class ContentTypeDefinitions
         [Export]
         [Name(ContentTypeNames.CSharpContentType)]
         [BaseDefinition(ContentTypeNames.RoslynContentType)]
+        // Adds the LSP base content type to ensure the LSP client activates on C# files.
+        // From Microsoft.VisualStudio.LanguageServer.Client.CodeRemoteContentDefinition.CodeRemoteBaseTypeName
+        // We cannot directly reference the LSP client package in EditorFeatures as it is a VS dependency.
+        [BaseDefinition("code-languageserver-base")]
```

Comment:
Uhhhh ya how did this work before lmao

Reply:
Intellicode!
https://devdiv.visualstudio.com/DevDiv/_git/IntelliCode-VS?path=%2Fsrc%2FVSIX%2FIntelliCode.VSIX%2FRefactorings%2FContentTypeDefinitions.cs&_a=contents&version=GBmain

Reply:
lol the hellllllll